public/apiSchema.json

```diff
@@ -1 +1,4 @@
nAuthorized"}}}},"404":{"description":"notFound","content":{"application/json":{"schema":{"$ref":"#/components/schemas/errors/notFound"}}}},"500":{"description":"badImplementation","content":{"application/json":{"schema":{"$ref":"#/components/schemas/errors/badImplementation"}}}}}}},"/users/:username":{"get":{"summary":"Get the details of user with provided id.","tags":["Users"],"security":[{"bearerAuth":[]}],"responses":{"200":{"description":"User details","content":{"application/json":{"schema":{"$ref":"#/components/schemas/users"}}}},"404":{"description":"notFound","content":{"application/json":{"schema":{"$ref":"#/components/schemas/errors/notFound"}}}},"500":{"description":"badImplementation","content":{"application/json":{"schema":{"$ref":"#/components/schemas/errors/badImplementation"}}}}}}},"/users/picture":{"post":{"summary":"Post user profile picture","consumes":["multipart/form-data"],"parameters":[{"in":"formData","name":"profile","type":"file","description":"Profile picture to upload"}],"tags":["Users"],"responses":{"200":{"description":"User image","content":{"application/json":{"schema":{"$ref":"#/components/schemas/users/img"}}}},"401":{"description":"unAuthorized","content":{"application/json":{"schema":{"$ref":"#/components/schemas/errors/unAuthorized"}}}},"404":{"description":"notFound","content":{"application/json":{"schema":{"$ref":"#/components/schemas/errors/notFound"}}}},"413":{"description":"entityTooLarge","content":{"application/json":{"schema":{"$ref":"#/components/schemas/errors/entityTooLarge"}}}},"415":{"description":"unsupportedMediaType","content":{"application/json":{"schema":{"$ref":"#/components/schemas/errors/unsupportedMediaType"}}}},"500":{"description":"badImplementation","content":{"application/json":{"schema":{"$ref":"#/components/schemas/errors/badImplementation"}}}}}}},"/wallet/":{"get":{"summary":"Gets the user wallet details","tags":["wallet"],"responses":{"200":{"description":"Return wallet","content":{"application/json":{"schema":{"$ref":"#/components/schemas/wallet"}}}},"401":{"description":"unAuthorized","content":{"application/json":{"schema":{"$ref":"#/components/schemas/errors/unAuthorized"}}}},"500":{"description":"badImplementation","content":{"application/json":{"schema":{"$ref":"#/components/schemas/errors/badImplementation"}}}}}}},"/wallet/:username":{"get":{"summary":"Gets the user wallet details for a username, if you are an authorized superuser","tags":["wallet"],"responses":{"200":{"description":"Return wallet","content":{"application/json":{"schema":{"$ref":"#/components/schemas/wallet"}}}},"401":{"description":"unAuthorized","content":{"application/json":{"schema":{"$ref":"#/components/schemas/errors/unAuthorized"}}}},"403":{"description":"forbidden","content":{"application/json":{"schema":{"$ref":"#/components/schemas/errors/forbidden"}}}},"500":{"description":"badImplementation","content":{"application/json":{"schema":{"$ref":"#/components/schemas/errors/badImplementation"}}}}}}}}}
```

Comment:
This line is repeated in the file. @MehulKChaudhari  Please check.

Reply:
We are not really using schema now ig.

Reply:
Fixing it. Thank you